platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -472,7 +478,18 @@ connectivity named-singleton BLE {
 	 * The name of the device as advertised via GAP.
 	 */
 	configuration-item deviceName : string
-	
+	/**
+	 * Boolean representing if the MAC address is to be configured
+	 */
+	configuration-item IsMacAddrConfigured : bool
+	/**
+	 * MAC Address to be configured. Unused if IsMacAddrConfigured is false
+	 */
+	configuration-item MacAddr : uint64
+	/**
+	 * BLE peripheral service type.
+	 */
+	required configuration-item Service : BLE_Service
```

Comment:
Please be more specific what exactly this configuration does. Either add more comments in the enum definition or explain it here.

Reply:
added documentation